source/crud/crud.md

```diff
@@ -176,18 +176,20 @@ interface Collection {
   /**
    * Finds the documents matching the model.
    *
-   * Note: The filter parameter below equates to the $query meta operator. It cannot
-   * contain other meta operators like $maxScan. However, do not validate this document
-   * as it would be impossible to be forwards and backwards compatible. Let the server
-   * handle the validation.
-   *
    * Note: result iteration should be backed by a cursor. Depending on the implementation,
    * the cursor may back the returned Iterable instance or an iterator that it produces.
    *
-   * @see https://www.mongodb.com/docs/manual/core/read-operations-introduction/
+   * @see https://www.mongodb.com/docs/manual/tutorial/query-documents/
```

Comment:
This is documentation for constructing a query filter. The more relevant docs for a driver developer would be the `find` command itself.

```suggestion
   * @see https://www.mongodb.com/docs/manual/reference/command/find/
```